fix(sandpack-tests): introduce `hideTestsAndSupressLogs` prop (#775)
codesandbox/sandpack: website/docs/src/pages/advanced-usage/components.mdx

`}
</CodeBlock>

### Hiding Tests

> You can hide the test files with `visibleFiles` prop.
> Additionally if you want to supress test content and only show the test results, you can use `hideTestsAndSupressLogs` option.
> This option will hide the test files and supress the console logs. And it will disable the verbose button.

Reviewer: ```suggestion
You can hide the test files using the `visibleFiles` prop. Additionally, if you want to suppress test content and only show the test results, you can use the `hideTestsAndSuppressLogs` option. This option will hide the test files, suppress the console logs, and disable the verbose button.
```

<CodeBlock stack>

> {`import { 

Reviewer: Can you lint/prettier this snippet?

Author: Sure! I ran it and fixed the issues you mentioned about linting & formatting. Also changed `hide` to `show` and removed `hideTestsAndSupressLogs` from context and added it to the test component. When I format it somehow messed up with tables, but just reverted back and added my change after.

  SandpackProvider, 
  SandpackLayout, 
  SandpackCodeEditor, 
  SandpackTests 
} from "@codesandbox/sandpack-react";

> 

Reviewer: Remove extra line breaks

 

export default () => (
  <SandpackProvider
    template="test-ts"
    options={{

>       hideTestsAndSupressLogs:true,

Reviewer: Missing a space between key/value

>        visibleFiles: ["/add.ts"],

Reviewer: Indent it correctly
